cmd/harmony/config.go

```diff
@@ -95,8 +95,12 @@ type txPoolConfig struct {
 }
 
 type pprofConfig struct {
-	Enabled    bool
-	ListenAddr string
+	Enabled            bool
+	ListenAddr         string
+	Folder             string
+	ProfileNames       []string `toml:",omitempty"`
```

Comment:
I think these values are critical and shall not be hidden from users even the value is empty. Do you think we shall remove the "omitempty" toml flag?